spec/utils.ts

```diff
@@ -37,18 +37,33 @@ export const wallet: Wallet = {
   },
 }
 
-export function buildURL(uri: string) {
+export function buildURL(uri: string, queryString?: Record<string, string>) {
```

Comment:
What if we do the same as in `createAuthHeaders` with the default parameter? You'll be able to return the string plus the query parameters.